src/pages/home/index.js

```diff
@@ -80,20 +80,18 @@ export function HomePage () {
             >
               <ul>
                 {
-                  attributes.map((attribute) =>
+                  attributes.map((attribute, index) =>
                     attribute &&
-                    <li key={attribute.nome}>
-                      <SwiperSlide className="swiperSliide " key={attribute.nome} >
-                        <div className='slide-container'>
-                          <h2 className="slidetitulo"> {attribute.nome} </h2>
-                          <div className='event-container'>
-                            <p className="descricao"> {attribute.descricao}</p>
-                          </div>
-                          <Link to='#' className='ver-mais'>ver mais sobre o evento</Link>
-                          <img src={urlCms + attribute?.foto_divulgacao?.data?.attributes?.url} className="slideimage" />
+                    <SwiperSlide className="swiperSliide " key={ index }>
```

Comment:
Existe alguma outra propriedade que seja unica em `attribute`? Evite usar index como key (veja a seção "Pitfall" no final desse artigo https://pt-br.react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key)

Reply:
Resolvido, não tinha nenhum id dentro do attributes então passamos ele como abjeto na request